test/resolvers/utils.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const { MongoMemoryServer } = require('mongodb-memory-server')
+const mongoose = require('mongoose')
+const Token = require('../../src/models/Token')
+const Domain = require('../../src/models/Domain')
+const { await } = require('signale')
+
+// Start MongoDB Instance
+const mongod = new MongoMemoryServer()
+
+// Create connection to mongoose before all tests
+exports.before = async t =>
```

Comment:
There's a new function in src/utils/connect that can be used here.

Reply:
Done